app/components/LiveDock.vue.ts

```diff
@@ -196,6 +188,13 @@ export default class LiveDock extends Vue {
     this.chatService.refreshChat();
   }
 
+  get hideChat() {
+    // previewEnabled is poorly named.  It really just means stuff
```

Comment:
Nitpick: use block comments

Reply:
I find normal comments easier than block comments.  Block comments take up more space and are annoying.